src/App.js

```diff
@@ -55,9 +55,21 @@ const PageRoute = ({ path, children }) => {
   )
 }
 
+// Authenticate for only applicants that have been accepted
 const AuthPageRoute = ({ path, children }) => {
-  const { isAuthed } = useAuth()
-  return <Route path={path}>{isAuthed ? <Page>{children}</Page> : <Redirect to="/login" />}</Route>
+  const { isAuthed, user } = useAuth()
+  if (!isAuthed) {
```

Comment:
this is probably a redundant check but should we include `!(isAuthed || user)` so L70 doesn't ever become undefined?

Reply:
Would changing line 70 to be `user?.status` work? (If I'm understanding your comment correctly) I feel a bit weird about including a check on `isAuthed` because if the user isn't auth'd then we'd just send them to `/`, which isn't very helpful.

Reply:
ah yes that would work too!